frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions.jsx

```diff
@@ -2,14 +2,21 @@
 import cx from "classnames";
 import PropTypes from "prop-types";
 import { Component, Fragment } from "react";
+import { t } from "ttag";
```

Comment:
If I scroll down to the very last suggestion and try to hover the icon, the suggestions list gets scrolled up:

https://github.com/metabase/metabase/assets/6830683/577a81dd-adbe-4e9b-9dac-92fc559eaac7

Reply:
I can't reproduce this on any browser. It seems like a scrolling bug in the popover element but I'm not seeing it.

Reply:
It didn't help :(

I wonder if it might be related to #38595